recipes-multimedia/picamera-libs/picamera-libs.bb

```diff
@@ -21,3 +21,6 @@ INHIBIT_PACKAGE_STRIP = "1"
 INHIBIT_SYSROOT_STRIP = "1"
 SOLIBS = ".so"
 FILES_SOLIBSDEV = ""
+
+COMPATIBLE_HOST = "null"
+COMPATIBLE_HOST:rpi = "(.*)"
```

Comment:
Isn't this supposed to be COMPATIBLE_MACHINE?

Reply:
Both work. Host is better since it may not mark the recipe machine specific when it is compiling

Reply:
But I guess they should be machine specific in this case, shouldn't they?

Reply:
It won’t matter but if you prefer then I would make the change

Reply:
hang on this recipe needs more love. Let me finish all and upload PR

Reply:
Looks good. Thanks and merged.